src/containers/Account/ProfilePage.js

```diff
@@ -242,28 +231,33 @@ function ProfilePage(props) {
                           id="phone-number"
                           name="phoneNumber"
                           placeholder="Phone Number"
+                          maxLength="250"
                           value={state.phoneNumber}
                           onChange={onChangeField}
                         />
                       </div>
                     </div>
                   </div>
-
-                  <div className="form-group">
-                    <button
-                      type="submit"
-                      className="btn btn-primary submit"
-                      disabled={isLoading}
-                    >
-                      {isLoading ? (
-                        <img src={loader} alt="" />
-                      ) : (
-                        'Update Profile'
-                      )}
-                    </button>
+                  <div className="row">
+                    <div className="col text-right ml-5">
+                      <button
+                            type="submit"
+                            className="btn btn-primary submit"
+                            disabled={isLoading}
+                          >
+                            {isLoading ? (
+                              <img src={loader} alt="" />
+                            ) : (
+                              'Update Profile'
+                            )}
+                        </button> 
+                    </div>
+                  </div>
+                  <div className="row">
+                    <div className="col">
+                      <Error error={error} />
+                    </div>
```

Comment:
Do we need wrap this?

Reply:
It helps keep consistent spacing and padding in bootstrap.

Not a hard requirement obviously but I'm gonna say yeah.